### ISSUE TITLE
Discrepancy with Google Translation

### ISSUE THREAD

**Issue by AdamElitzur** [AidanWelch/google-translate-api#43]
Hi Aidan, 

Loving the API so far!

I did have one issue, when I'm translating something from Hebrew to English, I'm getting a different translation from Google Translate. The one the API is giving me is incorrect, and the one Google Translate gives me is much better. Do you think I'm using it wrong, or have you seen inconsistencies with Google Translate's website?

This is my prompt from the translate call, which returns this result: "Aryan main, I would like to the mountains at the distance"
![image](https://github.com/AidanWelch/google-translate-api/assets/50723103/6dde59b2-b812-4c27-822d-fea466d036c0)
However, this is what Google translate returns: "I lift my head, I will lift my eyes to the mountains in the distance"

Any ideas about this? I've also been having this issue with some other languages, but this is one that I found right now. Thanks so much!

**Comment by AidanWelch** [comment]
I'm glad you're liking it!   My immediate guess would have been that it was because of `forceBatch` but you already set that to false so I'd have to investigate.  I'll try to take a look when I get a chance soon (If I take too long feel free to @ me)

**Comment by AdamElitzur** [comment]
Thanks so much! Let me know if you find anything, and if it's happening on your side or not.

**Comment by AdamElitzur** [comment]
@AidanWelch I hope all is well! Finding anything yet with the discrepancy? Anything else I should try in the meantime? Thanks so much.

**Comment by AidanWelch** [comment]
Well, the issue is with batch translation as
```
translate('ארים ראשי, אשא עיני אל ההרים במרחקים', {from: 'he', to: 'en', forceBatch: false, fallbackBatch: false}); // => res.text === 'I lift my head, I will lift my eyes to the mountains in the distance'
```

However that responds with `Error: Too Many Requests` sometimes.  So, it must just be more aggressive rate limiting.  With `fallbackBatch: true` when singleTranslate fails then batchTranslate is called rather than throwing an error.  But, to be honest, it might have been a mistake on my part to degrade translation quality rather than throw an error by default.

Basically, `forceBatch: false` and `fallbackBatch: false` must be set to force the more accurate singleTranslate.

**Comment by AidanWelch** [comment]
@AdamElitzur (Just pinging in case you didn't get the notification)

**Comment by AdamElitzur** [comment]
Thanks so much @AidanWelch! If I set forceBatch: false and fallbackBatch: false, then could that cause errors if there are too many requests?
I would like for a translation to show no matter what, even if the quality is worse, if that is the best possible outcome (better than an error). So actually, I like your move to degrade translation quality rather than throw an error by default! Thank you!

**Comment by AidanWelch** [comment]
@AdamElitzur Yep you got it exactly right!  I'm glad that solution works for you!
